tests/models/bloom/test_modeling_bloom.py

```diff
@@ -441,33 +446,6 @@ def test_batch_generation_padd(self):
             tokenizer.decode(greedy_output_without_pad[0, :-3], skip_special_tokens=True),
         )
 
-    @slow
-    def test_right_left_batched_input(self):
```

Comment:
Why would we delete this?

Reply:
Because we always use padding_side=left for batched generation for autoregressive models so for me there is no point keeping this test